common/djangoapps/student/models.py

```diff
@@ -1606,96 +1601,13 @@ def emit_event(self, event_name, enterprise_uuid=None, request=None):  # pylint:
             segment_traits['email'] = self.user.email
 
             if event_name == EVENT_NAME_ENROLLMENT_ACTIVATED:
```

Comment:
Call `course_enrollment_email.apply_async((self.user.id, str(self.course_id)))` this async task after this if condition.